File: Week1_Wed/JsonCData/JsonCData/ViewController.m
//
//  ViewController.m
//  JsonCData
//
//  Created by trainer on 1/6/16.
//  Copyright © 2016 trainer. All rights reserved.
//

#import "ViewController.h"
#import "PaperInfoManger.h"
@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    NSURLSession *session = [NSURLSession sharedSession];
    
    NSURL *url = [NSURL URLWithString:@"http://112.124.122.38/acountingExam/getPaperInfo.php"];
    
    NSURLRequest *request = [[NSURLRequest alloc] initWithURL:url];
    NSURLSessionDataTask *task = [session dataTaskWithRequest:request completionHandler:^(NSData * _Nullable data, NSURLResponse * _Nullable response, NSError * _Nullable error) {
        // NSString *receviedStr = [[NSString alloc] initWithData:data encoding:NSUTF8StringEncoding];
        //NSLog(@"%@",receviedStr);
        NSArray  *dataArr = [NSJSONSerialization JSONObjectWithData:data options:NSJSONReadingAllowFragments error:nil];
        
        [PaperInfoManger insertAllPapers:dataArr]; 
     
        
    }];
    [task resume];

    
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: Week2_Wed/JKAudioPlayer/JKAudioPlayer/ViewController.swift
//
//  ViewController.swift
//  JKAudioPlayer
//
//  Created by trainer on 1/9/16.
//  Copyright © 2016 trainer. All rights reserved.
//

import UIKit
import AVFoundation
class ViewController: UIViewController {
    
    var audioPlayer:AVAudioPlayer!
    
    override func viewDidLoad() {
        super.viewDidLoad()
       let audioUrl = NSBundle.mainBundle().URLForResource("TestSound", withExtension: "wav")
        
        
        do {
            audioPlayer = try AVAudioPlayer( contentsOfURL: audioUrl!)
        } catch {
            print(error)
        }
        
        
       
       
        audioPlayer.numberOfLoops = -1
        audioPlayer.volume = 1
        audioPlayer.currentTime = 0
        audioPlayer.play()
        
        
        
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

